### ISSUE TITLE
Odoo Enterprise - Can we move the chatter to the bottom?

### ISSUE THREAD

**Issue by JordiBForgeFlow** [odoo/odoo#67035]
Impacted versions: all in Odoo Enterprise

The user is forced to use the chatter on the right, on bigger screen sizes. This really irritates many users, specially when they are using a form view that contains a one2many with multiple fields. In those cases the user wants to make use the most of the screen.

OCA has https://github.com/OCA/web/tree/13.0/web_responsive, which provides a much more useful responsive UI.

**Comment by Yenthe666** [comment]
+1, loads of my customers has this behaviour 😄 
It would be cool to make it a user or company configuration perhaps?

**Comment by janbec** [comment]
Don't like it either, even on a 27" screen one2many fields quickly become scrollable.
I would like to have the option to have the chatter below the document while keeping the document viewer on the side (haven't figured out how to do that yet)

**Comment by pedrobaeza** [comment]
I prefer the chatter on the right, but it seems I'm lone in that opinion...

For me, a bad form design leads to require an horizontal scrollbar.

**Comment by JordiBForgeFlow** [comment]
Take a sales order form view and add two or three extra fields in the invoice line. Very easily you end up being almost unable to have space to add the product, as odoo resizes the columns.

**Comment by gdgellatly** [comment]
@JordiBForgeFlow I prefer it on the right. For no other reason than viewing attachments side by side with the document is really really useful. But I have lots of customers who complain too. 21:9 monitor aspect ratio or greater is the easy fix :) 

I think most of the problem is the new column resizing/squeezing algorithm to accomodate the optional fields attribute. Even fields who don't need it get 92px minimum and it squeezes RTL. So left hand side fields bear the brunt of it, in fact at some point it literally just says, I'm running out of space, divide what I have left by the number of fields.

For me a better option is possibly a drawer function that scrolls the chatter in and out, although it doesn't really solve the entire problem, because we still get this issue even on models without chatter, wizard modals etc. And that is the column squeezing algorithm. If only the fields supported relative widths - I think it is the cell_padding attribute, but it gets overridden.

**Comment by pedrobaeza** [comment]
Yes, document viewing is my killer feature for having it right. I agree since v13, the column spacing is less optimal.

**Comment by Cedric-Pigeon** [comment]
My customers also complain about this point. I installed the OCA addon where the chatter position can be defined per user.

**Comment by Yenthe666** [comment]
@Cedric-Pigeon could you link the OCA module here please? :)

**Comment by talentomarcos** [comment]
Hi everyone!

I've an issue with the chatter that when in small screens is on the bottom; but on big ones is on bottom and right side at the same time, this is only on one module and is used the same way everywhere.
Someone knows what is happening ?
Even if I do something in one is like working in the two at the same time
And when I try to make a change on one and this is happening, an Error is shown

Thanks a lot!
![Screen Shot 2024-02-15 at 12 25 29 (2)](https://github.com/odoo/odoo/assets/43836557/6197f9f1-6762-4115-9369-45c602e85bf0)
![Screen Shot 2024-02-15 at 12 25 22](https://github.com/odoo/odoo/assets/43836557/4036de71-2f9d-40a3-bc51-ca5be29ed8bb)

**Comment by milan-web9** [comment]
For anyone who needs a per-user workaround today—especially on Odoo Online/SaaS, where custom add-ons such as `web_responsive` generally cannot be installed—I built **Odoo Chatter Manager**, a Chrome extension specifically for this layout problem.

It adds a small floating control to Odoo form views. Each user can move the Chatter below the form, hide and show it instantly, or keep it on the right with an adjustable width. It can also remove the large empty margins so one2many fields and Sales Order lines have more horizontal space. The original Odoo layout can be restored immediately from the extension popup.

Everything runs only in the browser: the Odoo server, database and source code are not modified.

This does not replace the native feature request—a built-in per-user or company setting would still be preferable—but it provides an immediate personal option for the cases described in this issue.

[Odoo Chatter Manager](https://chromewebstore.google.com/detail/odoo-chatter-manager/iiebmiebejoacgmcldhndbhnlldjccol)
